apps/pesayetu/src/components/HURUmap/IndicatorTitle/Action.js

```diff
@@ -79,15 +79,18 @@ function Action({ children, header, icon, title, id, ...props }) {
       >
         {({ TransitionProps }) => (
           <Fade {...TransitionProps} timeout={350}>
-            <ClickAwayListener onClickAway={handleClose}>
-              <Paper className={classes.paper}>
-                <ButtonBase onClick={handleClose} className={classes.header}>
-                  <Typography className={classes.title}>{header}</Typography>
-                  <CloseIcon />
-                </ButtonBase>
-                {children}
-              </Paper>
-            </ClickAwayListener>
+            {/* Fix: https://github.com/mui/material-ui/issues/27154#issuecomment-1117386458  */}
+            <div>
+              <ClickAwayListener onClickAway={handleClose}>
+                <Paper className={classes.paper}>
+                  <ButtonBase onClick={handleClose} className={classes.header}>
+                    <Typography className={classes.title}>{header}</Typography>
+                    <CloseIcon />
+                  </ButtonBase>
+                  {children}
+                </Paper>
+              </ClickAwayListener>
+            </div>
```

Comment:
Can't this be a single reusable component? Seems like it's exactly the same in both apps and it's specific/related to share.